apps/website/src/content/docs/anchor.mdx

```diff
@@ -17,7 +16,19 @@ The anchor's `href` attribute is the destination URL while the text within the a
 
 ## Usage
 
-Consider using a named anchor when you want the user to land not only on a specific page, but also to arrive at a content area that may be below the fold so the user does not have to scroll to discover it.
+The `Anchor` component is a styled wrapper over the native `<a>` element, and it can be used in the same way as the native element.
+
+```jsx
+<Anchor href='https://example.com'>Example</Anchor>
+```
+
+Consider using a named anchor when you want the user to land not only on a specific page, but also to arrive at a content area that may be below the fold so the user does not have to scroll to discover it. Named anchors are created by adding an `id` attribute to an element, and then linking to that element using the `href` attribute.
```

Comment:
Is this needed? Not sure if this is iTwinUI specific.

Reply:
I was expanding on the text that was already there. I've shortened it since it's just regular anchor behavior. [`58d121f` (#1856)](https://github.com/iTwin/iTwinUI/pull/1856/commits/58d121f294999ce3dd0172f35a6bf9192373a680)